src/pages/search.vue

```diff
@@ -150,5 +150,22 @@ export default BrowsePage
 </script>
 
 <style lang="scss" scoped>
-@import '~/styles/results-page.scss';
+.search {
+  margin: 0;
+}
+.search-grid-ctr {
+  background-color: $color-wp-gray-0;
+  min-height: 600px;
+  padding: 0;
+
+  @include mobile {
+    width: 100%;
+    flex: none;
+  }
+}
+.grid-sidebar {
+  padding: 0;
+  border-right: 1px solid $color-transition-gray;
+  width: 21.875rem;
+}
```

Comment:
Can we change these to TW utils and drop the SCSS entirely?

Reply:
This page will need to change soon, so I would postpone it to another PR, @dhruvkb

Reply:
I thought the same as you, the redesign should definitely reduce the custom css to what is not included with Tailwind.